third_party/vault/deploy.sh

```diff
@@ -37,9 +37,6 @@ case "$1" in
     undeploy)
         undeploy
         ;;
-    wait_for_vault)
-      wait_for_vault
-      ;;
     *)
         echo "usage: $0 [deploy|undeploy|wait_for_vault]"
```

Comment:
wait_for_vault should be removed from usage instructions as well

Reply:
fixed